### PR TITLE
Translate kubernetes/qucikstart.md to zh #1109 Closes #1109

### LINKED ISSUE #1109: [Docs] Translate kubernetes/qucikstart.md to zh
### The MD file link you want to translate (must be in English) 
https://github.com/WasmEdge/WasmEdge/blob/master/docs/book/en/src/kubernetes/quickstart.md

### Target language
https://github.com/WasmEdge/WasmEdge/blob/master/docs/book/en/src/kubernetes/quickstart.md

### PR DESCRIPTION
**Pull request by javadoors**
Signed-off-by: paul.zhang <javadoors@126.com>

### CODE REVIEW

File: docs/book/zh/src/kubernetes/quickstart.md [hunk 1]
```diff
@@ -1 +1,54 @@
 # 快速开始
+
+我们为您创建了基于 Ubuntu 的脚本，以便您在标准 Kubernetes 设置中快速开始使用以下运行时组合。
```
Comment (alabulei1) [inline docs/book/zh/src/kubernetes/quickstart.md:3]: you 统一翻译成你吧

File: docs/book/zh/src/kubernetes/quickstart.md [hunk 2]
```diff
@@ -1 +1,54 @@
...
+
+| CRI（高级）runtime | OCI（低级）runtime | |
+| --- | --- | --- |
+| CRI-O | crun + WasmEdge | [脚本](https://github.com/second-state/wasmedge-containers-examples/blob/main/.github/workflows/kubernetes-crio.yml) |
+| containerd | crun + WasmEdge | [脚本](https://github.com/second-state/wasmedge-containers-examples/blob/main/.github/workflows/kubernetes-containerd.yml) |
+
+## CRI-O 和 crun
+
+在 Ubuntu 20.04 上您可以使用 CRI-O [install.sh](https://github.com/second-state/wasmedge-containers-examples/blob/main/crio/install.sh) 脚本来安装 CRI-O 和 `crun ` 。
```
Comment (alabulei1) [inline docs/book/zh/src/kubernetes/quickstart.md:12]: you 翻译成你

File: docs/book/zh/src/kubernetes/quickstart.md [hunk 3]
```diff
@@ -1 +1,54 @@
...
+$ wget -qO- https://raw.githubusercontent.com/second-state/wasmedge-containers-examples/main/kubernetes_crio/install.sh | bash
+```
+
+[simple_wasi_application.sh](https://github.com/second-state/wasmedge-containers-examples/blob/main/kubernetes_crio/simple_wasi_application.sh) 脚本展示了如何从 Docker Hub 拉取[ WebAssembly 应用程序](demo/wasi.md) ，然后在 Kubernetes 中将其作为容器化应用程序运行。
+
+```bash
+$ wget -qO- https://raw.githubusercontent.com/second-state/wasmedge-containers-examples/main/kubernetes_crio/simple_wasi_application.sh | bash
+```
+
+您应该会在控制台日志中看到 WebAssembly 程序打印的结果。[这里是一个例子](https://github.com/second-state/wasmedge-containers-examples/runs/4186005677?check_suite_focus=true#step:6:3007)。
```
Comment (alabulei1) [inline docs/book/zh/src/kubernetes/quickstart.md:30]: you 翻译成你

File: docs/book/zh/src/kubernetes/quickstart.md [hunk 4]
```diff
@@ -1 +1,54 @@
...
+$ wget -qO- https://raw.githubusercontent.com/second-state/wasmedge-containers-examples/main/kubernetes_containerd/install.sh | bash
+```
+
+[simple_wasi_application.sh](https://github.com/second-state/wasmedge-containers-examples/blob/main/kubernetes_containerd/simple_wasi_application.sh) 脚本展示了如何从 Docker Hub拉取[ WebAssembly 应用程序](demo/wasi.md)，然后在 Kubernetes 中将其作为容器化应用程序运行。
+
+```bash
+$ wget -qO- https://raw.githubusercontent.com/second-state/wasmedge-containers-examples/main/kubernetes_containerd/simple_wasi_application.sh | bash
+```
+
+您应该会在控制台日志中看到 WebAssembly 程序打印的结果。[这里是一个例子](https://github.com/second-state/wasmedge-containers-examples/runs/4577789181?check_suite_focus=true#step:6:3010)。
```
Comment (alabulei1) [inline docs/book/zh/src/kubernetes/quickstart.md:52]: you 翻译成 你

**Review by alabulei1 — COMMENTED** [review]
Thanks for your contribution! Only one thing needs to pay attention to.

File: docs/book/zh/src/kubernetes/quickstart.md (hunk 1 above)
Comment (javadoors) [reply docs/book/zh/src/kubernetes/quickstart.md:3]: OK,I will repair it.

**Comment by alabulei1** [comment]
LGTM
@hydai  Could you help merge this PR?

**Review by hydai — APPROVED** [review]

**Pull request merged**